packages/tools/src/tools/annotation/PlanarFreehandROITool.ts

```diff
@@ -578,6 +575,20 @@ class PlanarFreehandROITool extends AnnotationTool {
     return annotationsWithinSlice;
   }
 
+  protected getAnnotationStyle(context) {
+    // This method exists only because `super` cannot be called from
+    // _getRenderingOptions() which is in an external file.
+    return super.getAnnotationStyle(context);
+  }
+
+  protected getPolylinePoints(annotation: Annotation): Types.Point2[] {
+    // Combine code from renderContourBeingDrawn, renderClosedContourBeingEdited
+    // and renderOpenContourBeingEdited to return the polyline points that needs
+    // for the segments that need to be rendered. Currently they are being
+    // rendered by this class but it should be rendered by the base class.
```

Comment:
I'm still looking into this because this tools works in a totally different way compared to all other tools.